grunttasks/eslint.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc'
+    },
+    files: [
+      '{,bin/,config/,grunttasks/,lib/**/,test/,test/lib/mocks}*.js'
```

Comment:
![](http://cdn.ninjakiwi.com/thumbs/large/Globs2-lg.jpg)

Although, I am mildly curious about that "test/" trickery there and wonder why the glob isn't just "test/**/" (instead of "test/,test/lib/mocks").